src/routes/logout/+page.server.ts

```diff
@@ -0,0 +1,17 @@
+import type { Actions } from "./$types";
+
+export const actions = {
+  default: async ({ cookies, locals: { getKVClient } }) => {
+    const userToken = cookies.get("ldafUserToken");
+    if (!userToken) return { success: true };
+    cookies.delete("ldafUserToken", { sameSite: "none", path: "/" });
+    try {
+      const kvClient = await getKVClient();
+      await kvClient.deleteUserInfoByToken(userToken);
+    } catch (err) {
```

Comment:
So if I just navigate to `/logout` when I'm already logged in, it continues to say I'm logged in until  I click the "logout" button. However, it looks like the related redis values and user token cookies are getting deleted here. Are they getting instantiated elsewhere on page load? I'm trying to wrap my head around the difference between fully logging out and loading this page url by itself (not a likely path to occur, I know, but humor me).